packages/react-charting/src/components/LineChart/LineChartRTL.test.tsx

```diff
@@ -371,7 +408,7 @@ describe('Line chart - Subcomponent line', () => {
       expect(lines).toHaveLength(8);
     },
     undefined,
-    beforeAll,
+    undefined,
```

Comment:
You can omit the last two arguments since they are already optional and not followed by a defined argument.